integrations.json

```diff
@@ -1,4 +1,87 @@
 [
+  {
+    "name": "AspNetMvc",
```

Comment:
Can you regenerate this `integrations.json` file? I think this may have gotten mixed up with your AspNetMvc / WebApi PR?

Reply:
Good catch.
Took me a good while to understand why this was happening. Turns out that I GAC'd the assemblies when I was working on the ASP.NET MVC stuff.